pages/button-dropdown/icon-expandable.page.tsx

```diff
@@ -0,0 +1,92 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useState } from 'react';
+
+import ButtonDropdown, { ButtonDropdownProps } from '~components/button-dropdown';
+import SpaceBetween from '~components/space-between';
+
+import ScreenshotArea from '../utils/screenshot-area';
+
+import styles from './styles.scss';
+
+export const items: ButtonDropdownProps['items'] = [
+  {
+    id: 'category1',
+    text: 'category1',
+    iconName: 'gen-ai',
+    items: [...Array(2)].map((_, index) => ({
+      id: 'category1Subitem' + index,
+      text: 'Sub item ' + index,
+      iconName: 'star',
+    })),
+  },
+  {
+    id: 'category2',
+    text: 'category2',
+    iconName: 'settings',
+    items: [...Array(3)].map((_, index) => ({
+      id: 'category2Subitem' + index,
+      text: 'Cat 2 Sub item ' + index,
+    })),
+  },
+  {
+    id: 'item1',
+    text: 'Item 1',
+  },
+  {
+    id: 'category3',
+    text: 'category3',
+    iconName: 'folder',
+    items: [...Array(2)].map((_, index) => ({
+      id: 'category3Subitem' + index,
+      text: 'Sub item ' + index,
+    })),
+  },
+];
+
+export default function IconExpandableButtonDropdown() {
+  const [expandToViewport, setExpandToViewport] = useState(false);
```

Comment:
Is it useful to test this feature with `expandToViewport` set to `true` and `false`? If not, I would remove this and the corresponding input and not set this prop in the component.